mne/report.py

```diff
@@ -1158,6 +1271,17 @@ def parse_folder(self, data_path, pattern='*.fif', n_jobs=1, mri_decim=2,
             warnings.warn('`subjects_dir` and `subject` not provided.'
                           ' Cannot render MRI and -trans.fif(.gz) files.')
 
+    def view(self):
+        """Open a current verison of the html report in the web browser
+        """
+        tempdir = _TempDir()
+        fname = op.join(tempdir, 'report.html')
+        self.save(fname, open_browser=True)
+        # import webbrowser
+        # webbrowser.open_new_tab('file://' + fname)
```

Comment:
see how we do it here:

https://github.com/mne-tools/mne-python/blob/master/mne/viz/utils.py#L461